Assets/Scripts/WorldEngine/Terrain/TerrainCell.cs

```diff
@@ -290,6 +290,14 @@ public void SetAlteration(TerrainCellAlteration alteration)
         
         OriginalAltitude = alteration.OriginalAltitude;
         Altitude = alteration.Altitude;
+
+#if DEBUG
```

Comment:
Debug info is always useful :)

Reply:
Yeah, though sometimes debug blocks tends to clutter the code. Maybe I need to update my code editor to do stuff like auto hide debug blocks.